client/src/components/MoviesContainer.jsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import MoviesResults from './MoviesResults'
+import movies from './movies'
+import SearchContainer from './SearchContainer';
+import searchFilterType from './SearchFilter/searchFilterType';
+import sortType from './Sort/sortType'
+import BackToSearch from './BackToSearch'
+import MoviePage from './MoviePage'
+
+class MoviesContainer extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            list: [],
+            filter: searchFilterType.title,
+            sortType: sortType.releaseDate,
+            selectedMovie:null,
+            moviesOfTheSameGenre:[],
+            hasError:false
+        };
+    }
+
+    componentDidMount() {
```

Comment:
_trivial_
Actually, it is not criminal to use `componentDidMount` here, but it seems, that you can do sorting right inside where you declare the `state` - in the constructor